background.ts

```diff
@@ -245,7 +247,7 @@ Electron.app.whenReady().then(async() => {
 
     dockerDirManager.ensureCredHelperConfigured();
 
-    // Path management strategy will need to be selected after an upgrade
+    // Path management strategy will need to be selected after an update
```

Comment:
What is the difference between an update and an upgrade? The feature is called the "Upgrade Responder", but `upgrade` is renamed to `update`. Why does this matter.

Similarly there is an UpgradeResponderVersion, which may have an UpdateAvailable.

We should use either one term consistently, or define the difference between them.

Reply:
Regarding the difference in meaning between "update" and "upgrade", I like the answers given at [this stackexchange quesion](https://english.stackexchange.com/questions/13353/are-there-any-differences-between-update-and-upgrade). I like to think of updates as smaller improvements, whereas upgrades are big changes.

If we think that way, then we should be using the term "update" wherever we can. Unfortunately the name "Upgrade Responder" is fixed.

As I said in https://github.com/rancher-sandbox/rancher-desktop/pull/4062#discussion_r1175675773:

> I'm going to change the terminology to use Update unless it is in the context of Upgrade Responder (can't change that name)

I still think the best course of action is as I said above: use the term "update" everywhere except for where we are referring directly to Upgrade Responder. And if there is a comment we can add to describe this policy, we should do so. Though it's hard to see a best place for such a comment, since this terminology spans a lot of different parts of the code. Maybe multiple comments?